myuw/static/css/course_card.less

```diff
@@ -72,7 +72,12 @@ c8a #473EC5;
         padding: 2px 4px 2px 4px;
         border: 1px solid #eee;
     }
-
+    
+	.course-credit-type {
```

Comment:
@DannyNineNineNine What is this class for? I don't see it referenced anywhere or used.

Reply:
I'll check this when I'm back in the office on Thursday.